crates/vsock/src/vhu_vsock_thread.rs

```diff
@@ -705,7 +710,7 @@ mod tests {
         assert!(VhostUserVsockThread::epoll_register(
             epoll_fd,
             dummy_fd.as_raw_fd(),
-            epoll::Events::EPOLLOUT
+            epoll::Events::EPOLLOUT,
```

Comment:
There seem to be some artifacts from the reformat left here. Would either drop them or move to a separate commit.

Reply:
Removed, thanks!